types.go

```diff
@@ -220,16 +220,32 @@ func (p *Permission) Validate(vr *ValidationResults) {
 	}
 }
 
+// ResponsePermission can be used to allow responses to any reply subject
+// that is received on a valid subscription.
+type ResponsePermission struct {
+	MaxMsgs int           `json:"max"`
+	Expires time.Duration `json:"ttl"`
+}
+
+// Validate the response permission.
+func (p *ResponsePermission) Validate(vr *ValidationResults) {
+	// Any values can be valid for now.
+}
+
 // Permissions are used to restrict subject access, either on a user or for everyone on a server by default
 type Permissions struct {
-	Pub Permission `json:"pub,omitempty"`
-	Sub Permission `json:"sub,omitempty"`
+	Pub  Permission          `json:"pub,omitempty"`
+	Sub  Permission          `json:"sub,omitempty"`
+	Resp *ResponsePermission `json:"resp,omitempty"`
```

Comment:
I was thinking of this for import/export more than per user, looks good for per user to me.

Reply:
There will be something for export to show its a request for a stream, this is specifically for permissions of users that connect.